model_analyzer/result/result_comparator.py

```diff
@@ -63,17 +56,16 @@ def __init__(self,
             for i in range(len(gpu_metric_types))
         }
 
-        self._metric_priorities = metric_priorities
-        self._comparison_threshold_factor = (comparison_threshold_percent *
-                                             1.0) / 100
-
-        # TODO implement a weighted comparison
-        self._weights = weights
+        # Normalize metric objectives
```

Comment:
I would use `_metric_weights` as that is a more accurate description and purpose. `objectives` do not make sense here.